solc/CommandLineInterface.cpp

```diff
@@ -322,7 +322,7 @@ void CommandLineInterface::handleGasEstimation(string const& _contract)
 	if (eth::AssemblyItems const* items = m_compiler->assemblyItems(_contract))
 	{
 		Gas gas = GasEstimator::functionalEstimation(*items);
-		u256 bytecodeSize(m_compiler->runtimeObject(_contract).bytecode.size());
+		u256 bytecodeSize(m_compiler->object(_contract).bytecode.size());
```

Comment:
I think this was correct. The part after the `+` is the costs of code deployment. The cost of the transaction is not visible here at all.